infrastructure/app/chart/energy-comparison-table/values.yaml

```diff
@@ -17,6 +17,7 @@ secrets:
   # CONTENTFUL_ENVIRONMENT_ID: contentful-env-id
   # CONTENTFUL_SPACE_ID: contentful-space-id
   # CONTENTFUL_CONTENTFUL_CDA_TOKEN: contentful-cda-token
+  # SLACK_BOT_TOKEN: slack-bot-token
```

Comment:
I assume this isn't necessary now that you're using a GitHub Actions secret to post to Slack?

Reply:
I wasn't really sure if it was needed here, but the other secrets used in deployment are listed here too, and are stored in GitHub environment as secrets too, so I've put it here to be on the safe side.

Reply:
Oh wait, I think I understand. I thought that I needed to include the `SLACK_BOT_TOKEN` here for completeness, as this was a list of secrets that needed to be set for the project. But I think you're saying that, because `SLACK_BOT_TOKEN` isn't used by helm for the deployment, just by GitHub in a workflow, it doesn't belong in this list of secrets. Is that what you meant?

Reply:
Amended so that GH secrets are explicitly passed to helm as required, rather than passing all GH secrets to the helm deployment.